src/https_helper.rs

```diff
@@ -23,15 +21,13 @@ impl From<Method> for reqwest::Method {
 }
 
 pub(crate) async fn https(
-    client_config: &Arc<ClientConfig>,
     url: impl AsRef<str>,
     method: Method,
     body: Option<String>,
 ) -> Result<Response, HttpsRequestError> {
     let method: reqwest::Method = method.into();
-    let client_config: ClientConfig = client_config.as_ref().clone();
     let client = reqwest::ClientBuilder::new()
-        .use_preconfigured_tls(client_config)
+        .use_rustls_tls()
```

Comment:
this removes the ability to pass the client config, which is not good, we should wait until reqwest is updated and we can pass it along again

Reply:
looks like this is part of https://github.com/seanmonstar/reqwest/pull/2059